src/c++/perf_analyzer/infer_data_manager_base.cc

```diff
@@ -30,6 +30,88 @@
 
 namespace triton { namespace perfanalyzer {
 
+cb::Error
+InferDataManagerBase::GetInputData(
+    const std::string& name, const ModelTensor& tensor, int stream_id,
+    int step_id, std::vector<const uint8_t*>& data_ptrs,
+    std::vector<size_t>& byte_size)
+{
+  size_t max_count = tensor.is_shape_tensor_ ? 1 : batch_size_;
+  std::vector<int64_t> shape;
```

Comment:
I think vectors should be plural so when we read this further down in the code, we know its a container and not a singular shape tensor

Reply:
actually it is a single shape. The issue is the code's reliance on default types. Ideally this should be a typedef somewhere, but that is too big of a change for this story.